src/assets/release-notes-1.16.json

```diff
@@ -510,12 +550,12 @@
     "kinds": ["cleanup"],
     "sigs": ["api-machinery", "auth", "cluster-lifecycle", "testing"],
     "duplicate": true,
-    "release_version": "1.16.0-alpha.2"
+    "release_version": "1.16.0-beta.0"
   },
   "78477": {
     "commit": "f31d786927a7855b3dd2200d16e2b172661fb919",
-    "text": "Add Filter extension point to the scheduling framework.",
-    "markdown": "Add Filter extension point to the scheduling framework. ([#78477](https://github.com/kubernetes/kubernetes/pull/78477), [@YoubingLi](https://github.com/YoubingLi))\n\n  Courtesy of SIG Scheduling, and SIG Testing",
+    "text": "   Add Filter extension point to the scheduling framework.",
```

Comment:
@saschagrunert -- how did these spaces get pulled in?

Reply:
Via the tool, I created an follow-up PR regarding this: https://github.com/kubernetes/release/pull/870